effort_controllers/src/joint_group_effort_controller.cpp

```diff
@@ -42,7 +42,7 @@ template <class T>
 void forward_command_controller::ForwardJointGroupCommandController<T>::starting(const ros::Time& time)
 {
   // Start controller with 0.0 efforts
-  commands_.resize(n_joints_, 0.0);
+  commands_buffer_.readFromRT()->assign(n_joints_, 0.0);
```

Comment:
I am aware of this shortcoming, but it works for the current solution, as you already pointed out..
Initially I didn't want to use the RealtimeBuffer, but it is currently the best (and tested) solution we have..